src/components/modal/ModalStepper.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Stepper from "@mui/material/Stepper";
+import Step from "@mui/material/Step";
+import StepLabel from "@mui/material/StepLabel";
+
+type Props = {
+  step: number;
+};
+
+// label per ogni punto
+const steps = ["Welcome", "Prenota", "Scegli la data", "Dicci di più"];
+
+export const ModalStepper = ({ step }: Props) => {
+  return (
+    <Box sx={{ width: "100%" }}>
+      <Stepper activeStep={step}>
+        {steps.map((label, index) => {
+          return (
+            <Step key={index}>
+              <StepLabel sx={{ flexDirection: "column", gap: "4px" }}>
+                {label}
+              </StepLabel>
+            </Step>
+          );
+        })}
+      </Stepper>
+    </Box>
+  );
+};
```

Comment:
Non sarebbe meglio fargli accettare al component anche l'array di steps ? Così potremmo usarlo anche in altre parti dell'App.

Se questo array avesse il come type 
```
{
step: string
label: string
}[]
```

Reply:
@leonardodemarco00
Altra proposta che ti faccio: 
l'hook `useStep()` ritorna anche `stepIndex`. Che dici se inserissimo questo valore nel Provider così il questo component si gestisce tutta la logica all'interno e non accetta nessuna props ?

Reply:
Yes, forse però forse conviene passare in maniera separata step e l'array di labels, invece che avere un array di oggetti che contengono entrambe
@omardeangelis che ne dici?

Reply:
@leonardodemarco00 
Ci sono due soluzioni: 
1)  Passare come props al componente. 
´{
step: string,
steps: {
step: string
label: string
}[]
}´
E nell modal stepper fare un un find e cercare lo step a cui ci troviamo. 

2) Andare nel ModalContext e digli che accetta anche steps, e stepIndex in cui sono entrambi parametri opzionali. e passare questi parametri attaverso al Modal che li passa sempre al Provider.
Poi nel ModalStepper prendersi questi valori dal ModalContext e usarli.

Reply:
@leonardodemarco00 anche se basterebbe solo stepIndex al context e passare all modal Stepper un array di label come props e salvarci questo array in `feature/consulenze/utils/constants` e importarlo ovunque.